protocol/contracts/reserve/Cargo.toml

```diff
@@ -24,6 +24,14 @@ combinations = [
         "net_main",
         "neutron-astroport-usdc_noble",
     ], include-rest = false },
+    { tags = [
```

Comment:
I think either minimized, like other packages, or in separate tables, `[[package.metadata.cargo-each.combinations]]`, but not this abomination which takes the worst of both worlds.

Reply:
this was not introduced with this change ... will align it to the other packages